decidim/decidim
# Improve experience, usability giving proposals minimum votes number

#### :tophat: What? Why?
This PR is implementing a solution to improve participants' experience and usability by giving a minimum number of votes to proposals. The notifications with the voting rules will always be visible enough and my support may end up being counted with me being aware.

#### :pushpin: Related Issues
- Related to #?
- Fixes #13124 

#### Testing

1. Log in as administrator.
2. Enter the admin panel and navigate to a process with a proposals component with a vote limit per participant and minimum votes per user enabled.
3. See the process in the browser.
4. Navigate to proposals on the frontend.
5. See that there is a banner showing the voting rules.
6. Click on a proposal and see a notification displaying the remaining minimum votes.
7. Vote for the minimum number of proposals, and then you'll see a notification indicating that you have correctly given all the minimum votes.
8. When you have not voted for the minimum number of proposals and you click outside the proposal component, then you'll see a modal notifying you that you have pending votes to give.

### :camera: Screenshots
![image](https://github.com/user-attachments/assets/6ad3fb6c-6aca-4af8-94f8-29d4b696cc1a)
![image](https://github.com/user-attachments/assets/3f23fe23-55fc-48f2-8d7e-6699c73f3af1)
voting rules banner
![image](https://github.com/user-attachments/assets/464f85fc-f06b-4cd3-89ad-a8b41a3162a7)
voting rules dropdown
![image](https://github.com/user-attachments/assets/2fa237ce-07cc-411e-9be1-41a085a8b1cb)
![image](https://github.com/user-attachments/assets/2847e9b5-73f4-43b6-8687-d3a91ca5f214)
exit_modal
:hearts: Thank you!
